Kudu.Core/Deployment/Generator/SiteBuilderFactory.cs

```diff
@@ -194,6 +194,17 @@ private ISiteBuilder ResolveProject(string repositoryRoot, string targetPath, ID
             {
                 return DetermineProject(repositoryRoot, projects[0], perDeploymentSettings, fileFinder);
             }
+            
+            // Check for ASP.NET 5 project without VS solution or project
+            string projectJson;
+            if (AspNet5Helper.TryAspNet5Project(targetPath, out projectJson))
+            {
+                return new AspNet5Builder(_environment,
+                                           perDeploymentSettings,
+                                           _propertyProvider,
+                                           repositoryRoot,
+                                           projectJson);
+            }
 
```

Comment:
@suwatch @ahmelsayed  I think if don't do that the SLN is found first and then the Project is deemed as Website, even thought it is a ASP.NET 5 when you use the "Project" setting in the Azure portal.